src/core_plugins/vega/public/vega_editor_template.html

```diff
@@ -0,0 +1,75 @@
+<div class="vegaEditor" ng-controller="VegaEditorController">
+  <div
+    ui-ace="{
+      onLoad: aceLoaded,
+      useWrapMode: true,
+      advanced: {
+        highlightActiveLine: false
+      },
+      rendererOptions: {
+        showPrintMargin: false
+      },
+      mode: 'hjson'
+    }"
+    id="vegaAceEditor"
+    ng-model="vis.params.spec"
+    data-test-subj="vega-editor"
+  ></div>
+
+  <div id="editor_actions">
+    <span dropdown>
+      <button
+        id="vegaHelp"
+        class="editor_action"
+        dropdown-toggle href="#"
+        aria-label="Vega help"
+      >
+        <span class="kuiIcon fa-question-circle"></span>
+      </button>
+      <ul
+        class="dropdown-menu"
+        role="menu"
+        aria-labelledby="vegaHelp"
+      >
+        <li role="menuitem">
+          <a target="_blank" href="https://www.elastic.co/guide/en/kibana/master/vega-graph.html">
+            Kibana Vega Help
+          </a>
+        </li>
+        <li role="menuitem">
+          <a target="_blank" href="https://vega.github.io/vega-lite/docs/">
+            Vega-Lite Documentation
+          </a>
+        </li>
+        <li role="menuitem">
+          <a target="_blank" href="https://vega.github.io/vega/docs/">
+            Vega Documentation
+          </a>
+        </li>
+      </ul>
+    </span>
+    <span dropdown>
+      <button
+        id="vegaOptions"
+        class="editor_action"
+        dropdown-toggle href="#"
```

Comment:
Remove `href="#"`

Reply:
This was copied from https://github.com/elastic/kibana/blob/821bd96d321c07db61b85961463735cd8861eb9e/src/core_plugins/console/public/index.html#L18 . Will fix in Vega. Should it also be fixed in console?

Reply:
Created a PR https://github.com/elastic/kibana/pull/15947